.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python: 2.7
+sudo: false
+cache:
+  directories:
+  - eggs
+  - parts/node
+env:
+  matrix:
+    - PLONE_VERSION=4.3
+    - PLONE_VERSION=5.0
+#    - PLONE_VERSION=5.1
+before_install:
+ - virtualenv .
```

Comment:
@tomgross is there any benefit on creating a virtualenv? travis is supposed to keep things isolated